lib/bokken_web/views/organizer_view.ex

```diff
@@ -0,0 +1,21 @@
+defmodule BokkenWeb.OrganizerView do
+  use BokkenWeb, :view
+  alias BokkenWeb.OrganizerView
+
+  def render("index.json", %{organizers: organizers}) do
+    %{data: render_many(organizers, OrganizerView, "organizer.json")}
+  end
+
+  def render("show.json", %{organizer: organizer}) do
+    %{data: render_one(organizer, OrganizerView, "organizer.json")}
+  end
+
+  def render("organizer.json", %{organizer: organizer}) do
+    %{
+      id: organizer.id,
+      champion: organizer.champion,
+      user_id: organizer.user_id,
```

Comment:
To keep everything more consistent with the other roles that exist, I think that we don't need/should send the `user_id` because it is not necessary. We only give it to the user on the route /auth/me.  What do you think?